docs/examples/hello-world-fpm/composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "usabilla/docker-hello-world-fpm",
+    "description": "A simple Hello World for Usabilla PHP Docker images",
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Renato Mefi",
+            "email": "gh@mefi.in"
+        }
+    ],
+    "require": {
+        "php": "^7."
```

Comment:
Wouldn't `"php": "^7"` work in this case?

Reply:
typo, I want 7.3 :D